compiler/rustc_builtin_macros/Cargo.toml

```diff
@@ -21,6 +21,7 @@ rustc_index = { path = "../rustc_index" }
 rustc_lexer = { path = "../rustc_lexer" }
 rustc_lint_defs = { path = "../rustc_lint_defs" }
 rustc_macros = { path = "../rustc_macros" }
+rustc_middle = { path = "../rustc_middle" }
```

Comment:
Yea, please don't add this dependency to `rustc_builtin_macros`. Crates before it in the crate graph can use `unreachable!` and friends